eth2/beacon/state_machines/forks/serenity/block_validation.py

```diff
@@ -621,8 +622,9 @@ def validate_randao_reveal(randao_reveal: BLSSignature,
     if not is_randao_reveal_valid:
         raise ValidationError(
             f"RANDAO reveal is invalid. "
-            f"reveal={randao_reveal}, proposer_pubkey={proposer_pubkey}, "
-            f"message_hash={message_hash}, domain={domain}"
+            f"proposer_index={proposer_index}, proposer_pubkey={proposer_pubkey}, "
```

Comment:
having spent some time debugging this stuff -- i agree this is a nice addition :)